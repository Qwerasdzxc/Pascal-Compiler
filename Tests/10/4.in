8
-5 12 46 21 -5 1 5 9

