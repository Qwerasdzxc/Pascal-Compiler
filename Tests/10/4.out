12 46 
-5 21 -5 1 5 9 